docs/aquacomputer-d5next-guide.md

```diff
@@ -0,0 +1,27 @@
+# Aquacomputer D5 Next watercooling pump
+_Driver API and source code available in [`liquidctl.driver.aquacomputer`](../liquidctl/driver/aquacomputer.py)._
+
+## Initialization
+
+Initialization is not required. The pump sends a status HID report every second as soon as it's connected.
```

Comment:
If there's a chance initialization will be necessary for future devices (and there probably is), it's better to continue to recommend that the device be initialized, and only note that _currently_ it _might_ be a no-op.

Additionally, since `initialize` outputs the firmware version, there already is value in calling/executing it (especially when reporting bugs to us).  For the same reason, it also makes sense to show an example of it here as well.